### PR TITLE
Updated package.json

### PR DESCRIPTION
**Pull request by mhemesath**
- Add repository
- Fix license expression

### CODE REVIEW

**Comment by roryhardy** [comment]
👍

File: lib/src/index.js [hunk 1]
```diff
@@ -33,7 +33,6 @@ var JSONRPC = function () {
    * Initializes a JSONRPC instance..
    * @param {object} methods - The JSONRPC methods to handle.
```
Comment (Matt-Butler) [inline lib/src/index.js:34]: This file is generated with babel, how did these lines change without a change in the es6 code?

Comment (mhemesath) [inline lib/src/index.js:34]: I bumped the version of babel, it appears to have cleaned up its indentation logic a little.

**Comment by Matt-Butler** [comment]
+1

**Pull request closed without merging**